### PR TITLE
45 automatically attach uploaded files to model output

### PR DESCRIPTION
**Pull request by bschroeter**
Option to add model output id to the upload call, allowing faster workflow.

### CODE REVIEW

**Comment by bschroeter** [comment]
Part of the integration work.

File: meorg_client/cli.py [hunk 1]
```diff
@@ -109,19 +109,31 @@ def list_endpoints():
...
-    files = response.get("data").get("files")
-    for f in files:
-        click.echo(f.get("file"))
+    response = _call(client.upload_files, files=list(file_path), attach_to=attach_to)
+
+    # Different logic if we are attaching to a model output immediately
+    if not attach_to:
+        files = response.get("data").get("files")
+        for f in files:
+            click.echo(f.get("file"))
```
Comment (SeanBryan51) [inline meorg_client/cli.py:134]: Just curious, why not echo the list of files in the `attach_to == True` case?

**Review by SeanBryan51 — APPROVED** [review]
Looks good overall, just have a question around not printing the list of files uploaded when specifying `--attach_to`.

File: meorg_client/cli.py (hunk 1 above)
Comment (bschroeter) [reply meorg_client/cli.py:134]: Thanks @SeanBryan51, the reasoning is that outputting the file IDs does not indicate that the attach command actually worked, only that it successfully uploaded.

Regardless, it occurs to me that the call will simply fail with a non-zero exit status if the attachment fails, so I see no harm in adding the output of file ids.

I have just pushed a new commit for this, also adding the attachment code to the parallel version of the upload.

Tests passing, closing issue.

**Pull request merged**